packages/docs/.vuepress/config.js

```diff
@@ -18,6 +18,12 @@ module.exports = {
     require('./plugins/plugin-choices/index.js'),
     require('./plugins/plugin-micromodal/index.js')
   ],
+  markdown: {
```

Comment:
good find! My plan was to just hide it with CSS. Ill add this in to the new theme as well 😂